src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
-export interface Figure {
+function convertNumber(number: number): number {
+  return Math.trunc(number * 100) / 100;
+}
 
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
```

Comment:
```suggestion
  shape: 'triangle' | 'circle' | 'rectangle',
  color: 'red' | 'green' | 'blue',
```
Create types `Color` and `Shape` above and use them here and below